rasa/cli/x.py

```diff
@@ -112,10 +112,24 @@ def _overwrite_endpoints_for_local_x(
     from rasa.utils.endpoints import EndpointConfig
     import questionary
 
+    # Checking if endpoint.yml has existing url and wait time values set, if so give warning we are overwriting
+    # the endpoint.yml file.
+    custom_wait_time_pulls = endpoints.model.kwargs.get("wait_time_between_pulls")
```

Comment:
can we move this line to line 128? We are not needing the value here, so it makes more sense to get it right before we use it.